src/engine/client/cg_api.h

```diff
@@ -163,6 +163,12 @@ bool        trap_GetEntityToken( char *buffer, int bufferSize );
 std::vector<std::vector<Keyboard::Key>> trap_Key_GetKeysForBinds(int team, const std::vector<std::string>& binds);
 int             trap_Key_GetCharForScancode( int scancode );
 bool        trap_R_inPVS( const vec3_t p1, const vec3_t p2 );
+
+void trap_R_BatchInPVS(
+	const vec3_t origin,
+	const std::vector<std::array<float, 3>>& posEntities,
+	std::vector<bool>& inPVS );
```

Comment:
How about returning the result?

Reply:
What do you mean?

Reply:
Return the `vector<bool>` instead of writing into a reference.

Reply:
Wouldn't this produce an extra copy of the array?

Reply:
No. A return value can be automatically moved from a local variable.